src/Stacktrace.php

```diff
@@ -128,6 +128,10 @@ public function addFrame(string $file, int $line, array $backtraceFrame): void
             $line = (int) $matches[2];
         }
 
+        if ($backtraceFrame['class'] ?? null) {
+            $backtraceFrame['class'] = preg_replace('/0x[a-zA-Z0-9]+$/', '', $backtraceFrame['class']);
```

Comment:
The memory address is in hex format, so a stricter regex is `/0x[a-fA-F0-9]+$/`

Reply:
updated the regex